lib/usdUfe/ufe/Utils.cpp

```diff
@@ -378,6 +379,47 @@ SdfPath uniqueChildPath(const UsdStage& stage, const SdfPath& path)
 
     return path.ReplaceName(TfToken(uniqueName));
 }
+bool isMaterialsScope(const Ufe::SceneItem::Ptr& item)
+{
+    if (!item) {
+        return false;
+    }
+
+    // Must be a scope.
+    if (item->nodeType() != "Scope") {
+        return false;
+    }
+
+    // With the magic name.
+    if (item->nodeName() == defaultMaterialScopeName()) {
```

Comment:
This line changed to use new DCC function. MayaUsd will set it so behavior is the same.